sdk-setup/src/mb2

```diff
@@ -1194,6 +1261,84 @@ run_prepare() {
     return $rc
 }
 
+cmake_enable_host_paths() {
+    # Handle path changes for Qt Creator
+    if [[ -f "CMakeCache.txt"
```

Comment:
I would prefer implementing the guard on caller side. Similarly for cmake_disable_host_paths.